### DISCUSSION TITLE
You don't have any light themes?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#343] [Q&A] [answered]
I'm talking about the UI. If you are a night owl then the dark themes are for you. But if you are like me, sitting in an open space with full of sun light, then it's not.

**Comment by emeraldtip** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/91365d5e-0bc6-4101-8e36-1d4fe73eaf1d)
Wdym?

(Window -> UI Prefers Color Scheme -> Light 
and 
Document -> Syntax Color Scheme -> Github)

**Comment by SpartanJ** [comment]
To be more explicit:

https://github.com/user-attachments/assets/2ba2f079-ef5a-4bdb-a097-d21fd92dd41e

Themes are also very easy to customize / add new, check [here](https://github.com/SpartanJ/ecode?tab=readme-ov-file#customizations).
